### PR TITLE
Ruff: add and fix INP

### PR DESCRIPTION
**Pull request by kiblik**
Add and fix INP rule

https://docs.astral.sh/ruff/rules/#flake8-no-pep420-inp

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

File: docker/__init__.py
[the remark this answers is not in the document]
Comment (kiblik) [reply docker/__init__.py:1]: I add `noqa` to `docker/install_chrome_dependencies.py`.

File: dojo/importers/importer/__init__.py
Comment (Maffooch) [inline dojo/importers/importer/__init__.py:1]: This directory does not exist anymore

File: dojo/importers/reimporter/__init__.py
Comment (Maffooch) [inline dojo/importers/reimporter/__init__.py:1]: This directory does not exist anymore

**Review by Maffooch — CHANGES_REQUESTED** [review]

**Review by Maffooch — APPROVED** [review]

**Review by blakeaowens — APPROVED** [review]

**Pull request merged**